addons/snailmail/static/src/models/snailmail_notification_popover_content_view.js

```diff
@@ -0,0 +1,26 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'SnailmailNotificationPopoverContentView',
+    recordMethods: {
+        /**
+         * @private
+         * @returns {Message}
+         */
+        _computeMessage() {
+            return this.popoverViewOwner.messageViewOwnerAsSnailmailNotificationContent.message;
```

Comment:
Related ?

Reply:
Related only works with a single `.`, so can't use it